amy/workshops/filters.py

```diff
@@ -139,13 +139,13 @@ def filter(self, qs, value):
             return ordering
 
         # `value` is a list
-        if any(v in ["lastname"] for v in value):
+        if any(v in ["family"] for v in value):
             return ordering.order_by("family", "middle", "personal")
-        elif any(v in ["-lastname"] for v in value):
+        elif any(v in ["-family"] for v in value):
             return ordering.order_by("-family", "-middle", "-personal")
-        elif any(v in ["firstname"] for v in value):
+        elif any(v in ["personal"] for v in value):
             return ordering.order_by("personal", "middle", "family")
-        elif any(v in ["-firstname"] for v in value):
+        elif any(v in ["-personal"] for v in value):
             return ordering.order_by("-personal", "-middle", "-family")
```

Comment:
I also fixed this filter in here: https://github.com/carpentries/amy/pull/2335/files#diff-76c8f195f620772cf38ce87a84d610ada62278260c666e317c8387d8f274adf7

I would prefer my approach unless you find something wrong with it. I think the extra choices were intentionally set to `"lastname"` and `"firstname"`, as we don't have these fields in any model, and `NamesOrderingFilter` allows to filter by additional fields if provided.

Reply:
Your approach looks good, I'll revert these changes